lib/full-site-editing/edit-site-page.php

```diff
@@ -170,6 +170,14 @@ function gutenberg_edit_site_init( $hook ) {
 	wp_enqueue_script( 'wp-format-library' );
 	wp_enqueue_style( 'wp-edit-site' );
 	wp_enqueue_style( 'wp-format-library' );
+
+	if (
+		current_theme_supports( 'wp-block-styles' ) ||
```

Comment:
We have [this same condition](https://github.com/WordPress/gutenberg/blob/9f4f7d41b0129fc934951d0e5b4d2c45a1bf8b5c/lib/client-assets.php#L353) for loading the styles in the post editor. I wonder if we should refactor that to share the code?

Reply:
If we do refactor this I see a function in `WP_Theme_Config` being introduce here #29667